lessons/Algebra2/exploring-linear-relationships/langs/en-us/pages/model-your-own.adoc

```diff
@@ -0,0 +1,44 @@
+= Building Linear Models
+++++
+<style>
+.studentAnswerMedium { min-width: 10em !important; }
+.studentAnswerLong { min-width: 22em !important; }
+td { padding: 0 !important; }
+</style>
+++++
+
+For this page, you'll need to the @online-exercise{https://code.pyret.org/editor#share=10bWYM4rOq_ZshtX6_M-xf-bPbfz61rd1, State Demographics Starter File} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+@n Which two columns will explore? @fitbruby{20em}{}{x-axis} and @fitbruby{20em}{}{y-axis}.
+
+@n Fill in the code to make a scatter plot for those columns:
+
+@show{(code '(scatter-plot states-table "state" (?ANS ______________________) (?ANS ______________________)))}
+
+@n Pick two states to use for your first model: @fitb{10em}{} and @fitb{10em}{}.
+
+@n Based on these two points, define your model in Function and Pyret Notation:
+
+[cols="^1a, ^1a", options="header"]
+|===
+| Function Notation
+| Pyret Notation
+| @math{f(x) =} @fitb{12em}{}x + @fitb{12em}{}
+| @show{(code '(define (f x) (+ (* (?ANS ______) x) (?ANS ______))))}
+|===
+
```

Comment:
Beneath lines, add in "slope" and "y-intercept"

Reply:
Done